code42.json

```diff
@@ -190,19 +173,19 @@
                     "contains": [
                         "email"
                     ],
-                    "column_name": "username",
+                    "column_name": "Username",
                     "column_order": 0
                 },
                 {
                     "data_path": "action_result.parameter.departure_date",
                     "data_type": "string",
-                    "column_name": "departure_date",
+                    "column_name": "Departure Date",
```

Comment:
These seem to be all caps anyway, so the most important part is replacing underscores with spaces...

It's just a table output though.

Also, we have noticed that the table output seem to be driven off the output rather than the summary